lms/static/js/dashboard/track_events.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Track interaction with the student dashboard..
+ */
+
+var edx = edx || {};
+
+(function ($) {
+    'use strict';
+
+    edx.dashboard = edx.dashboard || {};
+
+    edx.dashboard.TrackEvents = function() {
+
+         var course_title_link = $(".course-title > a"),
+            course_image_link = $(".cover"),
+            enter_course_link = $(".enter-course"),
+            options_dropdown = $(".wrapper-action-more"),
+            course_learn_verified = $(".message-copy > a"),
+            find_courses_btn = $(".btn-find-courses");
+
+        // Track a virtual pageview, for easy funnel reconstruction.
+        window.analytics.page('student', 'dashboard' );
```

Comment:
You don't need to do this. There's other JS included in the dashboard template which takes care of tracking a pageview.